src/index.js

```diff
@@ -3,30 +3,79 @@ var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
 var API = 'https://rickandmortyapi.com/api/character/';
```

Comment:
En el problema 2 debíamos utilizar ECMAScript 6 por lo tanto ya no utilizamos 'var'

```suggestion
const API = `https://rickandmortyapi.com/api/character/`;
```